src/index.js

```diff
@@ -79,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = '',
+  javascriptKey = randomString(20),
```

Comment:
This is going to cause a problem.  If any client key is set (clientKey, javascriptKey, restAPIKey, dotNetKey) then all requests to the server must use a valid client key... What was the actual issue, something with the SDK initialization?